ui-react/packages/atlasmap/src/Atlasmap/AtlasmapProvider.tsx

```diff
@@ -104,6 +104,7 @@ export interface IAtlasmapProviderProps {
   baseJavaInspectionServiceUrl: string;
   baseXMLInspectionServiceUrl: string;
   baseJSONInspectionServiceUrl: string;
+  baseCSVInspectionServiceUrl: string;
```

Comment:
So this is a breaking change? We'll have to update Syndesis where we consume this to provide the right url then.

Reply:
As there's no connector leverages CSV support ATM, I think either way works in Syndesis - skip adding or internally add `DataShape.Kind = CSV` in UI and add CSV REST endpoint in server for future use. It's just an addition, won't be visible to Syndesis user unless any connector specify CSV kind in its DataShape. Maybe user can develop an extension leveraging it?
@gaughan WDYT?

Reply:
My comment was more about the typescript type. This way the url is required, and the Atlasmap component can't be consumed in Syndesis without providing one.

If it becomes optional, we have to deal with potentially undefined urls where we use this.

Reply:
Let's provide the URL then, it's just next to the others

Reply:
@riccardo-forina, I'm having trouble finding the right place to set it in Syndesis. https://github.com/syndesisio/syndesis/search?q=jsonInspectionServiceUrl&unscoped_q=jsonInspectionServiceUrl gives me nothing...

Reply:
Not your fault, [we haven't merged the PR yet](https://github.com/syndesisio/syndesis/pull/8316). I'll add the param when needed, don't worry!

Reply:
I see! Thanks!